doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -264,6 +358,19 @@ An example of a proof using cumulativity
 
    Inductive eq@{i} {A : Type@{i}} (x : A) : A -> Type@{i} := eq_refl : eq x x.
 
+.. coqtop:: none
+
+   Set Printing Universes.
+
+.. coqtop:: out
+
+   Print eq.
+
+The universe of :g:`eq` is irrelevant here, hence proofs of equalities can be
+seen as living in any other universe.
```

Comment:
It may be worth saying that there's an implicit side condition that the universe has to be big enough to fit `A`